File: TDCODER/Kỹ thuật lập trình 1/Chương 4 - Cấu trúc lựa chọn/KTLT1_B9C4.py
a = int(input())
b = int(input())
c = input()

ans = 0
if c == "/" and b == 0:
    print("Khong chia duoc")
elif c == "+":
    ans = a + b
    print(ans)
elif c == "-":
    ans = a - b
    print(ans)
elif c == "*":
    ans = a * b
    print(ans)
elif c == "/":
    ans = a / b
    print("%.1f" %ans)
else:
    print("Phep toan khong hop le")

File: TDCODER/Kỹ thuật lập trình 1/Chương 4 - Cấu trúc lựa chọn/KTLT1_B9C4.cs
using System;

public class Program {
    public static void Main(string[] args) {
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        char c = char.Parse(Console.ReadLine());
        
        if (c == '/' && b == 0) {
            Console.Write("Khong chia duoc");
        } else {
            switch (c) {
                case '+':
                    Console.Write(a + b);
                    break;
                    
                case '-':
                    Console.Write(a - b);
                    break;
                    
                case '*':
                    Console.Write(a * b);
                    break;
                
                case '/':
                    Console.Write("{0:0.0}", (double) a/b);
                    break;
                
                default:
                    Console.Write("Phep toan khong hop le");
                    break;
            }
        }
    }
}